charts/aws/values.yaml

```diff
@@ -0,0 +1,102 @@
+nameOverride: ""
+fullnameOverride: ""
+labels: {}
+scraper:
+  name: aws
+  
+connectionDetials:
+  connection: ''
+  accessKey:
+    name: ''
+    value: ''
+    valueFrom:
+      serviceAccount: ''
+      helmRef:
+        key: ''
+        name: ''
+      configMapKeyRef:
+        key: ''
+        name: ''
+      secretKeyRef:
+        key: ''
+        name: ''
+
+  secretKey:
+    name: ''
+    value: ''
+    valueFrom:
+      serviceAccount: ''
+      helmRef:
+        key: ''
+        name: ''
+      configMapKeyRef:
+        key: ''
+        name: ''
+      secretKeyRef:
+        key: ''
+        name: ''
+  region: ''
+  endpoint: ''
+  skipTLSVerify: ''
+  assumeRole: ''
+
+cloudtrail:
+  maxAge: ''
+  exclude: []
+
+compliance: true
+
+# Scrape cost reports and match them with catalog items
+costReporting:
+  enabled: false
+  database: athena_database_name
+  region: athena_database_region
+  s3BucketPath: s3_bucket_path_for_athena_query_results
+  table: athena_database_table_name
+
+inventory: true
+
+patchDetails: true
+
+patchStates: true
+
+trustedAdvisorCheck: false
+
```

Comment:
Also, keeping all the keys in root, is it okay that way or should everything be prefixed with `scraper.` ?